tests/buyers/views/test_buyers.py

```diff
@@ -315,7 +392,7 @@ def test_copy_brief_and_redirect_to_copied_brief_edit_title_page(self):
     def test_copy_brief_for_expired_framework_redirects_to_edit_page_for_new_framework(self):
         self.data_api_client.get_brief.return_value = api_stubs.brief()  # dos1 brief
 
-        new_brief = self.brief  # dos2 brief
+        new_brief = self.brief.copy()  # dos2 brief
```

Comment:
So the functionality will mean we're using the old brief as the new brief here. Which isn't really the point of the copy function.
Putting the copy here in the test indicates what we're trying to achieve slightly better IMO.

Perhaps if we called self.brief something like self.brief_data that would be more indicative of what's going on?